apis/github/write/tool.gpt

```diff
@@ -0,0 +1,20 @@
+Name: GitHub
+Description: Provides full access to the GitHub API
+Share Credential: github.com/gptscript-ai/gateway-oauth2 as github.write with GITHUB_TOKEN as env and GitHub as integration and "repo" as scope
+Share Tools: list_issues from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: list_prs from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: create_issue from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: modify_issue from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: delete_issue from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: search_issues from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: create_pr from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: modify_pr from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: delete_pr from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: add_comment_to_issue from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: add_comment_to_pr from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: get_issue from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: get_pr from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: get_issue_comments from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: get_pr_comments from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: list_repos from github.com/njhale/tools/apis/github/code@enhance/github-api
+Share Tools: close_issue from github.com/njhale/tools/apis/github/code@enhance/github-api
```

Comment:
nit: For this one, you can do `tools: * from github.com/gptscript-ai/tools...` since it includes all of them.

Reply:
done